src/rules/no-unused-modules.ts

```diff
@@ -444,13 +444,47 @@ const fileIsInPkg = (file: string) => {
 export interface Options {
   src?: string[]
   ignoreExports?: string[]
-  missingExports?: true
+  missingExports?: boolean
   unusedExports?: boolean
```

Comment:
Maybe we should change it to be `true` to align the json schema instead?

Reply:
https://github.com/un-ts/eslint-plugin-import-x/pull/410/files#diff-fad22462aa10e51b14c769d122db4a6d1919c11856d5246ecbdd78344ad19b30L505

We should align all the schemas I think?

Reply:
To be honest I would probably set the type as boolean. So it should be possible for someone to set it to false even if it is the default